sass/includes/_header.scss

```diff
@@ -1,9 +1,48 @@
 .tna-header {
+  padding: 1rem 3rem 0rem 3rem;
```

Comment:
I don't think you need the `rem` unit on the `0` here. `padding: 1rem 3rem 0 3rem` should be fine.